appserver/admingui/corba/src/main/resources/sslPrepare-copy.inc

```diff
@@ -0,0 +1,82 @@
+<!--
+
+    DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+
+    Copyright (c) 2009-2017 Oracle and/or its affiliates. All rights reserved.
+
+    The contents of this file are subject to the terms of either the GNU
+    General Public License Version 2 only ("GPL") or the Common Development
+    and Distribution License("CDDL") (collectively, the "License").  You
+    may not use this file except in compliance with the License.  You can
+    obtain a copy of the License at
+    https://oss.oracle.com/licenses/CDDL+GPL-1.1
+    or LICENSE.txt.  See the License for the specific
+    language governing permissions and limitations under the License.
+
+    When distributing the software, include this License Header Notice in each
+    file and include the License file at LICENSE.txt.
+
+    GPL Classpath Exception:
+    Oracle designates this particular file as subject to the "Classpath"
+    exception as provided by Oracle in the GPL Version 2 section of the License
+    file that accompanied this code.
+
+    Modifications:
+    If applicable, add the following below the License Header, with the fields
+    enclosed by brackets [] replaced by your own identifying information:
+    "Portions Copyright [year] [name of copyright owner]"
+
+    Contributor(s):
+    If you wish your version of this file to be governed by only the CDDL or
+    only the GPL Version 2, indicate your decision by adding "[Contributor]
+    elects to include this software in this distribution under the [CDDL or GPL
+    Version 2] license."  If you don't indicate a single choice of license, a
+    recipient has the option to distribute your version of this file under
+    either the CDDL, the GPL Version 2 or to extend the choice of license to
+    its licensees as provided above.  However, if you add GPL Version 2 code
+    and therefore, elected the GPL Version 2 license, then the option applies
+    only if the new code is made subject to such option by the copyright
+    holder.
+
+-->
+
+<!-- sslPrepare-copy.inc  is based on sslPrepare.inc
+     After the REST issue 22134 is fixed we should change sslEdit.jsf to include the sslPrepare.inc -->
+
+setPageSessionAttribute(key="edit" value="#{false}" )
+mapPut(map="#{pageSession.createMap}", key="certNickname",  value="testName")
+gf.restRequest(endpoint="#{pageSession.createSslUrl}", attrs="#{pageSession.createMap}", method="POST", result="#{pageSession.resultMap}", throwException="false");
+if ("#{pageSession.resultMap.data.exit_code}=FAILURE") {
+    setPageSessionAttribute(key="edit" value="#{true}" )
+}
```

Comment:
This is still trying to create a new SSL without user's knowledge.  
A better approach is to try to get the SSL attributes and then save it.  If the save succeed, we know SSL exist, and we are not changing any of it's value since we are just saving it back.   If the save failed, we know we need to create it later when the user click the 'ok' button.

All you need to do is just copy the code from the sslButtons.inc that does the save and put it as a test here.